python_examples/sim_speeds/sim_runner_spatial.py

```diff
@@ -0,0 +1,84 @@
+# Code to run simulations and plot runtime for populations of varying size
+# but with constant number of population (10 individuals) initially infected.
+
+import os
+import logging
+# import matplotlib.pyplot as plt
+import time
+import numpy as np
+
+import pyEpiabm as pe
+
+# Setup output for logging file
+logging.basicConfig(filename='sim.log', filemode='w+', level=logging.DEBUG,
+                    format=('%(asctime)s - %(name)s'
+                            + '- %(levelname)s - %(message)s'))
+
+# Set config file for Parameters
+pe.Parameters.set_file(os.path.join(os.path.abspath(''),
+                       "spatial_parameters.json"))
+
+
+# Method to set the seed at the start of the simulation, for reproducibility
+
+pe.routine.Simulation.set_random_seed(seed=42)
+
+pop_sizes = np.array([int(1000), int(2000), int(4000), int(8000),
```

Comment:
Same, not sure if all the ints are needed

Reply:
Agree.
ints removed